src/helpers/redisHelper.ts

```diff
@@ -0,0 +1,9 @@
+import redis from 'redis';
+const client = redis.createClient();
+
+export const saveJWT = (key:string, value:string) => {
```

Comment:
We can reduce this to one line of code too, such as 
`export const saveJWT = (key: string, value: string) => client.set(key, value, redis.print)`
Same thing for the one below